### PR TITLE
Adding support of kube-ui

### PR DESCRIPTION
**Pull request by remoe**
interested to add kube-ui ? :)

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -102,6 +102,7 @@ DNS_UPSTREAM_SERVERS = ENV['DNS_UPSTREAM_SERVERS'] || "8.8.8.8:53,8.8.4.4:53"
 
 SERIAL_LOGGING = (ENV['SERIAL_LOGGING'].to_s.downcase == 'true')
 GUI = (ENV['GUI'].to_s.downcase == 'true')
+PLUGIN_KUBE_UI = ENV['PLUGIN_KUBE_UI'] || true
```
Comment (pires) [inline Vagrantfile]: Please, set default to `false`.

**Comment by remoe** [comment]
ok, done.

**Comment by pires** [comment]
@remoe can you add instructions on how to enable this to `README.md`?

**Comment by pires** [comment]
When done with it, please squash your commits into one and push so I can merge.

**Pull request closed without merging**